apps/apple/FluentUITester/macos/FluentUITester-macOS/FRNViewController.m

```diff
@@ -11,12 +11,18 @@ @interface FRNViewController () <RCTBridgeDelegate>
 
 @implementation FRNViewController
 
-- (void)loadView {
-  // The delegate must conform to RCTBridgeDelegate to retrieve the JS source file for the packager
+- (void)viewDidLoad {
+  [super viewDidLoad];
+  
   RCTBridge *bridge = [[RCTBridge alloc] initWithDelegate:self launchOptions:nil];
   RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge moduleName:@"FluentUITester" initialProperties:nil];
 
-  [self setView:rootView];
+  NSView *view = [self view];
+
+  [view addSubview:rootView];
+  [rootView setBackgroundColor:[NSColor windowBackgroundColor]];
+  [rootView setFrame:[view bounds]];
+  [rootView setAutoresizingMask:(NSViewMinXMargin | NSViewMinXMargin | NSViewMinYMargin | NSViewMaxYMargin | NSViewWidthSizable | NSViewHeightSizable)];
```

Comment:
Should be loadView. viewDidLoad is additional construction after loading from a xib. loadView is when you're making the hierarchy yourself.

Reply:
putting it in loadView gave me an infinite loop of trying to initialize the view. I'll see if I can fic that up

Reply:
Don't call[self view]. that's what attempts to lazily load the view. loadView should construct the hierarchy and then call [self setView:]